app/services/post_alerter.rb

```diff
@@ -468,7 +468,7 @@ def create_notification(user, type, post, opts = {})
     end
 
     # Make sure the user can see the post
```

Comment:
Might want to change this comment... or just remove it :) guardian name is expressive enough to me